gladier_xpcs/tools/__init__.py

```diff
@@ -13,6 +14,7 @@
 
 __all__ = [
     'TransferFromClutchToTheta',
+    'SourceTransfer',
```

Comment:
We don't really need TransferFromClutchToTheta no, and you're right it would make sense to remove it since there isn't really a theta ep to use anymore. I think it would also make sense to remove the other two flows too, since they're not pretty outdated. Can we track that in a separate issue?